packages/core/lib/command-utils.js

```diff
@@ -0,0 +1,209 @@
+const TaskError = require("./errors/taskerror");
+const { bundled, core } = require("../lib/version").info();
+const OS = require("os");
+const analytics = require("../lib/services/analytics");
+const { extractFlags } = require("./utils/utils"); // contains utility methods
+const globalCommandOptions = require("./global-command-options");
+const debugModule = require("debug");
+const debug = debugModule("core:command:run");
+const commands = require("./commands/commands");
+
+const getCommand = (inputStrings, options, noAliases) => {
```

Comment:
Couple things:
- Can we turn this into a single options argument?
- Is there a better name than `getCommand`? Or maybe this just needs a docstring explaining the function

Reply:
So this function returns the command name, its options, its meta info, and the run method for that command. Is there a better name that comes to mind for this stuff? I don't mind if they are named, I'll change them.